Week-1/Homework/1.Shortest-word.js

```diff
@@ -0,0 +1,23 @@
+//find the shortest word
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function shortestWord(array) {
+  if (typeof array != "object") {
```

Comment:
what if you passed an actual object into your function? `shortestWord({test: true, tite: true})` this will pass

Reply:
edi true tite mo

Reply:
nice